processor/spanmetricsprocessor/processor.go

```diff
@@ -371,6 +376,17 @@ func (p *processorImp) collectCallMetrics(ilm pmetric.ScopeMetrics) {
 	}
 }
 
+// collectIngestLatencyMetrics sets up a gauge metric for ingest latency and
+// populates it with the contents of p.ingestLatencyTimestamp.
+func (p *processorImp) collectIngestLatencyMetrics(ilm pmetric.ScopeMetrics) {
```

Comment:
```suggestion
func (p *processorImp) collectIngestLatency(ilm pmetric.ScopeMetrics) {
```
first off, its one metric not metrics, second allow me to pedantically say metric is implied

Reply:
I'm fine changing it, should note that I was trying to stay in line [with package style](https://github.com/Shopify/opentelemetry-collector-contrib/blob/5d8f42f361eba57c0f78eb390b1dbec28b291f97/processor/spanmetricsprocessor/processor.go#L311-L313):

```go
	p.collectIngestLatencyMetrics(ilm)
	p.collectCallMetrics(ilm)
	p.collectLatencyMetrics(ilm)
```

Reply:
i would say its a latency metric, stuff it in that method and execute if the config opt is set

Reply:
edfd0226a0d9554c360a3c96c4d1406113184911